backend/api/app/application/services/user_service.py

```diff
@@ -76,3 +107,152 @@ def verifySSID(self, ssid: str, ip_address: str):
             return True
         else:
             return False
+        
+    def logout_user(self, ssid: str, ip_address: str):
+            try:
+                if not ssid:
+                    return {"message": "SSID is required", "status": 400}
+
+                try:
+                    ses_id = uuid.UUID(ssid)
+                except ValueError:
+                    return {"message": "Invalid SSID format", "status": 400}
+
+                session = SessionRepository.get_active_by_id(ses_id, ip_address)
+                if not session:
+                    return {"message": "Session not found", "status": 404}
```

Comment:
When there is no session, that means that user is not logged in, also this should be handled by middleware and user should not worry if there is a session or not, he/she just needs to know that they're not logged in, which is also not 404 error.